src/mlpack/methods/kmeans/elkan_kmeans_impl.hpp

```diff
@@ -87,14 +90,13 @@ double ElkanKMeans<DistanceType, MatType>::Iterate(const arma::mat& centroids,
       // No change needed.  This point must still belong to that cluster.
       counts(assignments[i])++;
       newCentroids.col(assignments[i]) += arma::vec(dataset.col(i));
-      continue;
     }
     else
     {
       for (size_t c = 0; c < centroids.n_cols; ++c)
       {
-        // Step 3: for all remaining points x and centers c such that c != c(x),
-        // u(x) > l(x, c) and u(x) > 0.5 d(c(x), c)...
```

Comment:
No need to change this comment (I imagine it is because of the extra indentation that later got reverted with the custom OpenMP reductions); can you revert it to the original?